File: UNIDADE/flutter_application_finalProject/lib/cadastro/usuario.dart
class Usuario {
  static final Map<String, String> _usuarios = {
    'abc': '123',
  };

  static void adicionarUsuario(String login, String senha) {
    _usuarios[login] = senha;
  }

  static bool validarUsuario(String login, String senha) =>
      _usuarios[login] == senha;
      
}

File: UNIDADE/apiProjetoSensor/src/main/java/com/example/apiProjeto/model/Usuario.java
package com.example.apiProjeto.model;

import jakarta.persistence.*;
import jakarta.validation.constraints.NotBlank;

@Entity
@Table (name = "usuarioestufa")
public class Usuario {

    @Id
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    private Long id;

    @NotBlank(message = "O login é obrigatório")
    private String login;

    @NotBlank(message = "A senha é obrigatória")
    private String senha;

    public Usuario() {}

    public Usuario(String login, String senha) {
        this.login = login;
        this.senha = senha;
    }

    // Getters e Setters
    public Long getId() { return id; }
    public String getLogin() { return login; }
    public void setLogin(String login) { this.login = login; }
    public String getSenha() { return senha; }
    public void setSenha(String senha) { this.senha = senha; }
}

